ui/src/plugins/vumc/biovu.tsx

```diff
@@ -0,0 +1,256 @@
+import InfoIcon from "@mui/icons-material/Info";
+import FormControl from "@mui/material/FormControl";
+import MenuItem from "@mui/material/MenuItem";
+import OutlinedInput from "@mui/material/OutlinedInput";
+import Select, { SelectChangeEvent } from "@mui/material/Select";
+import Tooltip from "@mui/material/Tooltip";
+import Typography from "@mui/material/Typography";
+import { CriteriaPlugin, registerCriteriaPlugin } from "cohort";
+import Checkbox from "components/checkbox";
+import {
+  Filter,
+  FilterType,
+  makeArrayFilter,
+  UnaryFilterOperator,
+} from "data/filter";
+import { useUpdateCriteria } from "hooks";
+import produce from "immer";
+import GridLayout from "layout/gridLayout";
+import React from "react";
+import { CriteriaConfig } from "underlaysSlice";
+
+enum SampleFilter {
+  NONE = "None",
+  ANY = "Any",
+  ONE_HUNDRED = "100",
+  FIVE_HUNDRED = "500",
+}
+
+const sampleFilterDescriptions = {
+  [SampleFilter.NONE]: "Any SD Records (not requiring BioVU resources)",
+  [SampleFilter.ANY]: "Any BioVU DNA (no minimum amount or concentration)",
+  [SampleFilter.ONE_HUNDRED]: "At least 100ng with at least ng/µL BioVU DNA",
+  [SampleFilter.FIVE_HUNDRED]: "At least 500ng with at least ng/µL BioVU DNA",
```

Comment:
Doesn't need to be this PR, but could you try and match these strings in a `search` function so this shows up in the across-all-criteria search?

Reply:
Done, no reason not to support that. It does bring up the consideration that there are no rollup counts for these options. Not particularly important at the moment but worth considering how we might solve that.